kolibri/core/logger/api.py

```diff
@@ -90,6 +97,21 @@ def update(self, request, *args, **kwargs):
                 default_response[field] = method(instance)
         return Response(default_response)
 
+    def create(self, request, *args, **kwargs):
+        try:
+            return super(LoggerViewSet, self).create(request, *args, **kwargs)
+        except IntegrityError:
```

Comment:
interesting - like a "create or get" command?

Reply:
Pretty much - except because the id is calculated from the Model, we just return the model id from an instance of the model.